.travis.yml

```diff
@@ -50,8 +50,11 @@ after_success:
   # upload the documentation from the build with stable (automatically only
   # actually runs from the master branch, not individual PRs), not on OSX.
   - |
-      if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-        cd rusoto && travis-cargo --only stable doc-upload && cd ..
+      if [[ "$TRAVIS_OS_NAME" == "linux" && "$TRAVIS_RUST_VERSION" == "stable" ]]; then
+        cd rusoto && echo '<meta http-equiv=refresh content=0;url=rusoto_core/index.html>' > target/doc/index.html \
```

Comment:
We should `mkdir target/doc/rusoto/` and `echo '<meta http-equiv=refresh content=0;url=../rusoto_core/index.html>' > target/doc/rusoto/index.html` to catch existing links to http://rusoto.github.io/rusoto/rusoto/index.html such as the [crates.io page.](https://crates.io/crates/rusoto)